.gitignore

```diff
@@ -2,9 +2,10 @@
 /build/*
 /dist/*
 /RosettaSciIO.egg-info/*
+home/
 *__pycache__*
 *test_compilers.obj
 *pyd
 .spyproject/*
-rsciio/bruker/unbcf_fast.c
-.idea
\ No newline at end of file
+rsciio/bruker/unbcf_fast.c*
```

Comment:
Why is this change necessary?

Reply:
Without the wildcard, the `rsciio/bruker/unbcf_fast.cpython-310-x86_64-linux-gnu.so` file produced from `pip install -e .[dev]` is not ignored by git.

Reply:
I see and even if this is working, I would suggest to use the pattern suggested in https://github.com/hyperspy/rosettasciio/issues/85#issuecomment-1419706822, because they map the pattern to the file type to be filtered more explicitly:
There are two types of be ignore:
- the c code `unbcf_fast.c`
- and the corresponding binary `unbcf_fast.so` on unix or `unbcf_fast.pyd` on windows

I don't think we should depend on the way python name the compiled file (`unbcf_fast.cpython-310-x86_64-linux-gnu`) as the `cpython` part could change easily but the file extension (`so` or `pyd`) should be stable.

Reply:
I've used the pattern you refer to. So sorry for missing this in the first place.